### ISSUE TITLE
Ankiconnect error when exporting

### ISSUE THREAD

**Issue by OllTeeP** [arianneorpilla/jidoujisho#415]
![Screenshot_20241028-183410_jidoujisho.png](https://github.com/user-attachments/assets/1408b0b8-e224-4e27-b7e8-d3b31bf99ca1)

**Comment by forkyspoony** [comment]
Same issue here

**Comment by OllTeeP** [comment]
> Same issue here
> Edit: Tried updating to 2.9 preview 3, same thing

same

**Comment by avc1657** [comment]
Have you tried reinstalling jidoujisho, updating ankidroid to latest version, trying exporting a card with just one dictionary for testing, etc?

**Comment by MaximeSaland** [comment]
I also have the exact same problem with 2.9 preview 3 (using Google pixel 7)

**Comment by avc1657** [comment]
Weird. Im on Android 10, Ankidroid 2.19 and Jidoujisho 2.8.10. No issues here.

**Comment by abaga129** [comment]
I had the same issue and after reinstalling multiple times and also updating and then reinstalling Anki, I found that the issue was actually just that I had not selected a deck on the card creation screen in Jidoujisho.  After setting it to my mining deck, everything started working.
Also this app is absolutely incredible!

**Comment by OllTeeP** [comment]
> I had the same issue and after reinstalling multiple times and also updating and then reinstalling Anki, I found that the issue was actually just that I had not selected a deck on the card creation screen in Jidoujisho.  After setting it to my mining deck, everything started working.
> Also this app is absolutely incredible!

this fixed it for me thanks

**Comment by branko623** [comment]
> I had the same issue and after reinstalling multiple times and also updating and then reinstalling Anki, I found that the issue was actually just that I had not selected a deck on the card creation screen in Jidoujisho. After setting it to my mining deck, everything started working. Also this app is absolutely incredible!

This did it for me too, i was very confused after reinstall. Perhaps throw an exception with "no deck selected" message would be good?

**Comment by OllTeeP** [comment]
> > I had the same issue and after reinstalling multiple times and also updating and then reinstalling Anki, I found that the issue was actually just that I had not selected a deck on the card creation screen in Jidoujisho. After setting it to my mining deck, everything started working. Also this app is absolutely incredible!
> 
> This did it for me too, i was very confused after reinstall. Perhaps throw an exception with "no deck selected" message would be good? 

yeah agree

**Comment by thisguy1328** [comment]
got stuck on this too why is the drop down box invisible

**Comment by l0f3n** [comment]
Yeah i agree, it was not that intuitive. Working fine after selecting a deck.
